docs/installing.rst

```diff
@@ -5,42 +5,28 @@ Installation
 System Requirements
 *******************
 
-ThreatResponse now requires python >= 3.4.  It may still work(ish) with Python 2.7 but is not recommended due
-to security problems with cryptography libraries in Python 2.7.
+ThreatResponse requires python >= 3.4.
```

Comment:
Nit.  Should we add the corresponding code to raise an error for this condition on run pointing to a doc?

Reply:
It will throw an error because the shebang in bin/cli is set to python3 now.

Reply:
is there another way?